spikingjelly/activation_based/triton_kernel/flex_sn_inductor/hop.py

```
@@ -524,51 +144,50 @@ def eager_scan(
+# autograd graph which is chained via ``torch.stack``/indexing, giving a
+# full BPTT graph. AOTAutograd (``aot_function`` / ``make_fx``) traces this
+# graph natively by unrolling; see module docstring.
+flex_sn_scan.py_impl(torch._C.DispatchKey.Autograd)(eager_scan)
+
+
+def _patch_dynamo_hop_make_for_flexsn():
+    try:
+        from torch._dynamo.variables.higher_order_ops import (
+            TorchHigherOrderOperatorVariable,
+        )
+    except BaseException:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `BaseException`

(BLE001)